internal/tlsutil/tlsutil_test.go

```diff
@@ -0,0 +1,91 @@
+package tlsutil_test
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/kopia/kopia/internal/clock"
+	"github.com/kopia/kopia/internal/tlsutil"
+)
+
+func TestGenerateServerCertificate(t *testing.T) {
+	ctx := context.Background()
+	certValid := 24 * time.Hour
+	names := []string{"127.0.0.1", "localhost"}
+
+	cert, priv, err := tlsutil.GenerateServerCertificate(ctx, 2048, certValid, names)
+	if err != nil {
```

Comment:
we generally use require library to avoid if statements like that

require.NoError(t, err)
require.NotNil(t, cert)

and so on...

Reply:
Done.